packages/data/src/world/mm/snowhead_temple.yml

```diff
@@ -206,10 +206,10 @@
   exits:
     "Snowhead Temple Map Room Upper": "true"
     "Snowhead Temple Center Level 2 Dual": "true"
-    "Snowhead Temple Center Level 3 Iced": "has_mask_goron || can_hookshot || short_hook_anywhere || has_hover_boots" # todo hover boots trick
+    "Snowhead Temple Center Level 3 Iced": "has_mask_goron || can_hookshot || short_hook_anywhere || has_hover_boots" # Hovers trick not necessary until other settings call for it
     "Snowhead Temple Snow Room": "small_keys_sh(3)"
     "Snowhead Temple Fire Arrow": "true"
-    "Snowhead Temple Center Level 4": "short_hook_anywhere && (can_use_fire_short_range || has_mask_blast || has_bombchu)"
+    "Snowhead Temple Center Level 4": "short_hook_anywhere && (can_use_fire_arrows || has_mask_blast || has_bombchu)"
```

Comment:
Is this shooting the snowballs with fire arrows and then hookshotting to the area? does din's not reach?

Reply:
din's doesn't reach no. i tried at the very corner.